### ISSUE TITLE
Use modular Firebase admin in v1.x

### ISSUE THREAD

**Issue by kmjennison** [gladly-team/next-firebase-auth#488]
**Is your feature request related to a problem? Please describe.**

We should take advantage of modular Firebase admin:
https://firebase.google.com/docs/admin/migrate-node-v10#use-modules-instead-of-namespaces

**Describe the solution you'd like and how you'd implement it**

Currently, we do:
```js
import * as admin from 'firebase-admin'
```

We should use:
```js
import { initializeApp } from 'firebase-admin/app'
```

**Is this a breaking change?**
Would this require existing users to change how they're using `next-firebase-auth`?

[Edited] Yes, `firebase-admin` would need to be v10+.

**Describe alternatives you've considered**

None

**Comment by louisgv** [comment]
Firebase just released v11: https://github.com/firebase/firebase-admin-node/releases/tag/v11.0.0

I will try to use it. If there's any issue, should I report them here or should I make a new one? @kmjennison

**Comment by kmjennison** [comment]
@louisgv Please open a separate issue if you have a problem with v11. Thanks!

**Comment by kmjennison** [comment]
Closed in #548.
